extensions/amp-fit-text/0.1/amp-fit-text.js

```diff
@@ -79,13 +79,14 @@ class AmpFitText extends AMP.BaseElement {
   }
 
   /** @override */
-  prerenderAllowed() {
+  isRelayoutNeeded() {
     return true;
   }
 
   /** @override */
-  isRelayoutNeeded() {
-    return true;
+  prerenderCallback() {
```

Comment:
Yeah, it's fine. We can also split the difference and put it in the third method and call it from both places ;)